### PR TITLE
Added a help section with contents of the help documents

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Added a help section with links to view the markdown files in the `help` directory.

# Need Review By (Date)
- asap

# Urgency
- high

# Steps to Test
1. pull the latest from the stack branch & `composer install -n`
1. `composer require SU-HKKU/cardinal_service_profile:dev-help-ui`
1. `blt drupal:sync --site=cardinalservice`
1. login and view the page `/admin/help`
1. click on one of the links in the top "Cardinal Service" section
1. validate you see an empty page. (contents of the help documents coming soon)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: modules/cardinal_service_profile_helper/src/Controller/Documentation.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+   *
+   * @param string $document
+   *   File name of the help document.
+   *
+   * @return string
+   *   Page title.
+   */
+  public function getTitle($document) {
+    $html = $this->getDocumentation($document);
+    return $html['#title'];
```
Comment (joegl) [inline modules/cardinal_service_profile_helper/src/Controller/Documentation.php:50]: Will there's always be a title or should there be a check for a condition?

Comment (pookmish) [reply modules/cardinal_service_profile_helper/src/Controller/Documentation.php:50]: There should always be one, but null values don't throw errors anyways.

Comment (joegl) [reply modules/cardinal_service_profile_helper/src/Controller/Documentation.php:50]: Nevermind -- I see where this comes from now.

File: modules/cardinal_service_profile_helper/src/Controller/Documentation.php [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+   * @return array
+   *   Help markdown markup converted to html.
+   */
+  public function getDocumentation($document) {
+    // Invalid path.
+    if (!file_exists(self::getProfilePath() . "/help/$document.md")) {
+      throw new NotFoundHttpException($this->t('Document @document not found', ['@document' => $document]));
+    }
+    $html = MarkdownExtra::defaultTransform(file_get_contents($this->getProfilePath() . "/help/$document.md"));
+    $h1 = preg_grep('/<h1>.*?<\/h1>/', explode("\n", $html));
```
Comment (joegl) [inline modules/cardinal_service_profile_helper/src/Controller/Documentation.php:31]: Never seen preg_grep() function before neat.

**Pull request merged**